pkg/devfile/adapters/kubernetes/component/commandhandler.go

```diff
@@ -49,9 +49,8 @@ func (a *adapterHandler) Execute(devfileCmd devfilev1.Command) error {
 			switch status {
 			case remotecmd.Starting:
 				// Creating with no spin because the command could be long-running, and we cannot determine when it will end.
-				s.Start(fmt.Sprintf("Executing the application (command: %s)", devfileCmd.Id), true)
+				log.SpinnerNoSpin(fmt.Sprintf("Executing the application (command: %s)", devfileCmd.Id))
 			case remotecmd.Stopped, remotecmd.Errored:
-				s.End(status == remotecmd.Stopped)
```

Comment:
If there is an error a few seconds/minutes after the command has started, users would never be aware of this, I think. See my comment in https://github.com/redhat-developer/odo/pull/5964#pullrequestreview-1047724907